src/agent/rustjail/src/container.rs

```diff
@@ -989,13 +990,47 @@ impl BaseContainer for LinuxContainer {
             child_stdin = unsafe { std::process::Stdio::from_raw_fd(pseudo.slave) };
             child_stdout = unsafe { std::process::Stdio::from_raw_fd(pseudo.slave) };
             child_stderr = unsafe { std::process::Stdio::from_raw_fd(pseudo.slave) };
+
+            if let Some(proc_io) = &mut p.proc_io {
+                if let Some(mut stdin_stream) = proc_io.stdin.take() {
+                    let mut term_master = unsafe { File::from_raw_fd(pseudo.master) };
+                    let logger = logger.clone();
+                    tokio::spawn(async move {
+                        let res = tokio::io::copy(&mut stdin_stream, &mut term_master).await;
+                        debug!(logger, "copy from stdin to term_master end: {:?}", res);
+                        std::mem::forget(term_master); // Avoid auto closing of term_master
```

Comment:
I had noticed that in tty mode, the runtime side didn't send the close io rpc request, and here you "std::mem::forget(term_master)", thus the tty master would closed, which mean the container process's stdin didn't  close. So was it right to close the term master here?